packages/react/src/__tests__/jest-matcher.test.tsx

```diff
@@ -1,13 +1,6 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { styled } from '@compiled/react';
-import { toHaveCompiledCss } from '../matchers';
```

Comment:
Had to move this into the react package has it was causing a cyclic dependency and TS project references weren't happy.